index.js

```diff
@@ -22,14 +21,22 @@ const exec = (cmd, args) => {
 	).filter(Boolean);
 };
 
+const install = (bin, useYarn) => () => execa(bin, ['install'])
```

Comment:
Is the first time I use `execa`, so there might be a better way of doing this :upside_down_face: 

Also I would like to display a message, when falling back to `npm`, any suggestions on how @sindresorhus @SamVerschueren ?

Something like:
```bash
 ✔ Prerequisite check
 ✔ Git
 ✔ Cleanup
 ⠦ Installing dependencies using Yarn
     Yarn not installed, falling back to npm
 ✔ Running tests
 ✔ Bumping version
 ✔ Publishing package
 ✔ Pushing tags
```